JsonFormReadme.md

```diff
@@ -0,0 +1,71 @@
+
+-----------------------------------------------------------------------------------------------
+# documentation  
+
+## rules for create a JSON file
+
+1) data is written inside the curly brace.
+
+2) data is represented as key value pairs.
+
+3) key should be enclosed in double quotes .
+
+4) key and value should be seperate by colon (:).
+
+5) comma is used to seperate the data.
+
+6) square bracket hold arrays.
+
+7) curly braces hold objects.
+ 
+
+# required attributes
+
+1) key - a key is always string and enclosed in quotation mark
+
+2) name - specify the name of a feild
+
+3) label - used to create caption for a form control
+
+4) id - specify the unique id for an element
+
+5) type - defines the feild
+
+6) placeholder - shows information belongs in a feild
+
+7) validation - this value is a function that return a string errort or null if the feild is valid
+
+8) validation server - it needs to validate on the server in addtion to on the client
+
+9) class -
+
+10) wrapper class - Its provide all the of the componnets attribute and properties as props on
+
+the react component
+
+11) Onchange - it detect when the value of an imput element changes
+
+12) mandatory -
+ 
+
+## validation
+
+1) required - this keeps a list of required properties 
+
+2) minimum - this is the constraint to be put on the values and represent minimum acceptable value
+
+3) maximum - this is the constraint to be put on the value and represent the maximum acceptable value 
+
+4) multiple of - it is a result division of the instance by this keyword's value is an intege
+
+## field type 
+
+1) string - String  must written in double quotes 
```

Comment:
@sajana1829 These are not valid field types in the Json Form plugin... where did you get these `string` `number` `boolean` from? If you understood the functionality of the plugin you will see that field types are like `text`, `password`, `email` etc.
If you did not understand what you are documenting then you should ask from me or from Raghu, Hari, Yasika who are all working on this plugin.